data/mods/gen1/moves.ts

```diff
@@ -268,30 +268,39 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		},
 	},
 	disable: {
-		inherit: true,
```

Comment:
You should keep inherit: true and just override functions / other properties as necessary. You have a lot of extra duplicate code!